src/CMakeLists.txt

```diff
@@ -157,11 +159,9 @@ endif()
 
 if(USE_THREADS)
 	if(NOT WIN32)
+		set(THREADS_PREFER_PTHREAD_FLAG TRUE)
```

Comment:
TODO: what is this?

Reply:
CMake advices to use set this variable to TRUE :

https://cmake.org/cmake/help/latest/module/FindThreads.html

To be honnest I don't know why adding the flag `-pthread` is better but it seems encouraged by CMake.